src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -169,6 +169,71 @@ impl<T: AstInfo> AstDisplay for Statement<T> {
 }
 impl_display_t!(Statement);
 
+/// A static str for each statement kind
+pub fn statement_kind_label_value(kind: StatementKind) -> &'static str {
```

Comment:
Can this be `impl std::fmt::Display StatementKind` instead (maybe not a good idea, not sure).

Reply:
I could add a fmt::Display that uses this function but I need something that returns a `&str` for prometheus's with_label_values. Otherwise I think i'd be stuck creating a temporary string every time